walletlibrary/src/test/java/com/microsoft/walletlibrary/networking/operations/FetchOpenID4VCIRequestNetworkOperationTest.kt

```diff
@@ -0,0 +1,54 @@
+package com.microsoft.walletlibrary.networking.operations
+
+import com.microsoft.walletlibrary.did.sdk.datasource.network.apis.HttpAgentApiProvider
+import com.microsoft.walletlibrary.util.http.httpagent.IResponse
+import io.mockk.coEvery
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.runBlocking
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+
+class FetchOpenID4VCIRequestNetworkOperationTest {
+    private val expectedCredentialOffer = """
+        {
+            "credential_issuer": "metadata_url",
+            "issuer_session": "request_state",
+            "credential_configuration_ids": [
+                "credential_id"
+            ],
+            "grants": {
+                "authorization_code": {
+                    "authorization_server": "authorization_server"
+                }
+            }
+        }
+    """.trimIndent()
+
+    @Test
+    fun fetchOpenID4VCIRequestNetworkOperationTest_withTwoVPTokens_SucceedsToResolvePresentationRequestContent() {
```

Comment:
is this title correct?

Reply:
Will you add test cases for failures?

Reply:
Added a few failure cases.

Reply:
The title of the test case seems wrong

Reply:
Fixed the name as well. For some reason, it didn't show up automatically.